imagetagger/imagetagger/images/migrations/0017_imageset_zip_state.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.0.9 on 2018-11-19 17:13
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('images', '0016_settag_test'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='imageset',
+            name='zip_state',
+            field=models.IntegerField(choices=[(0, 'ready'), (1, 'processing'), (2, 'invalid')], default=2),
```

Comment:
Does not seem logical; why is 0 ready? Especially, 0 would be a more logical default than 2 (we are in an app/database world here, not shell ;-) ).